x-pack/test/detection_engine_api_integration/security_and_spaces/group10/risk_engine_install_resources.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const es = getService('es');
+
+  describe('install risk engine resources', () => {
+    it('should install resources on startup', async () => {
```

Comment:
It would be great to have a test where these resources already exist on startup!

Reply:
Do you know if we already have such tests? I am not sure that we can do that right now.

Reply:
Here's an example of a test that [loads a legacy signals index](https://github.com/elastic/kibana/blob/main/x-pack/test/detection_engine_api_integration/security_and_spaces/group10/create_signals_migrations.ts#L55-L57). 

We don't have the ability to load index templates or ILM via the `es_archiver` service, but you could certainly do that manually with the `es` service. There are lots of integration tests that use `esClient.indices.putIndexTemplate` etc.